src/mds/CDir.cc

```diff
@@ -2123,6 +2125,9 @@ void CDir::_omap_fetched(bufferlist& hdrbl, map<string, bufferlist>& omap,
       last_name = std::string_view(k_it->c_str(), n_key.name.length());
       null_keys.emplace_back(std::move(n_key));
       ++k_it;
+
+      if (!(++count % 1000))
```

Comment:
Any particular reason to check **1000** or **2000** (or in multiple of that) ?
Can we use Macro for these numbers? Just in case if we need to update these values in future so we can only update macro.

Reply:
For some code it may take a longer time for each loop so will use `1000` as others do in `mds/`, and for some code it will be faster so will use `2000` instead.

Sounds good. Will add one macro for the whole `mds/`.